src/BenchmarkDotNet/Toolchains/MonoAotLLVM/MonoAotLLVMGenerator.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using BenchmarkDotNet.Extensions;
+using BenchmarkDotNet.Helpers;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Portability;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains;
+using BenchmarkDotNet.Toolchains.CsProj;
+using BenchmarkDotNet.Toolchains.DotNetCli;
+using Microsoft.DotNet.PlatformAbstractions;
+
+
+namespace BenchmarkDotNet.Toolchains.MonoAotLLVM
+{
+    public class MonoAotLLVMGenerator : CsProjGenerator
+    {
+        private readonly string CustomRuntimePack;
+        private readonly string AotCompilerPath;
+        private readonly MonoAotCompilerMode AotCompilerMode;
+
+        public MonoAotLLVMGenerator(string targetFrameworkMoniker, string cliPath, string packagesPath, string customRuntimePack, string aotCompilerPath, MonoAotCompilerMode? aotCompilerMode)
+            : base(targetFrameworkMoniker, cliPath, packagesPath, runtimeFrameworkVersion: null)
+        {
+            CustomRuntimePack = customRuntimePack;
+            AotCompilerPath = aotCompilerPath;
+            AotCompilerMode = aotCompilerMode ?? MonoAotCompilerMode.mini;
+        }
+
+        protected override void GenerateProject(BuildPartition buildPartition, ArtifactsPaths artifactsPaths, ILogger logger)
+        {
+            BenchmarkCase benchmark = buildPartition.RepresentativeBenchmarkCase;
+            var projectFile = GetProjectFilePath(benchmark.Descriptor.Type, logger);
+
+            string useLLVM = "false";
+
+            if (AotCompilerMode == MonoAotCompilerMode.llvm)
+            {
+                useLLVM = "true";
+            }
```

Comment:
this can be simplified

```suggestion
            string useLLVM = AotCompilerMode == MonoAotCompilerMode.llvm ? "true" : "false";
```